support-models/src/main/scala/com/gu/support/redemptions/RedemptionData.scala

```diff
@@ -2,24 +2,23 @@ package com.gu.support.redemptions
 
 import cats.syntax.functor._
 import com.gu.support.encoding.Codec
-import com.gu.support.redemptions.redemptions.{CorporateAccountId, RedemptionCode}
 import io.circe.syntax._
 import io.circe.{Decoder, Encoder}
 
 sealed abstract class RedemptionData(val redemptionCode: RedemptionCode)
 
-case class CorporateRedemption(override val redemptionCode: RedemptionCode, corporateAccountId: CorporateAccountId) extends RedemptionData(redemptionCode)
+case class CorporateRedemption(override val redemptionCode: RedemptionCode) extends RedemptionData(redemptionCode)
```

Comment:
I'm wondering whether we need this ADT or whether all of them can be modelled with a single value class.  Depends if we're going to use the same dynamo table really.

Reply:
Maybe not but let's leave it for now until we're sure.